flow/scripts/open.tcl

```diff
@@ -51,7 +51,9 @@ proc read_timing {input_file} {
 
 if {[env_var_equals GUI_TIMING 1]} {
   puts "GUI_TIMING=1 reading timing, takes a little while for large designs..."
-  read_timing $input_file
+  if {[catch {read_timing $input_file} errMsg]} {
+      puts "Error reading timing: $errMsg"
+  }
 }
 
```

Comment:
Above the catch will report the error and continue execution, below we open the GUI, which we want to do even in the event of an error in read_timing, which is why the catch and continue above is what is needed.

Note that openroad -gui returns non-zero exit code, even when there are errors in the script.

If the user wants to run a script and have a non-zero exit code, then use `make run RUN_SCRIPT=foo.tcl` instead of `make open/gui_`